ui/packages/catalog-admin-module-sources/src/main/webapp/js/model/Source.js

```diff
@@ -33,18 +33,28 @@ define([
         Source.Model = Backbone.Model.extend({
             configUrl: "../jolokia/exec/org.codice.ddf.ui.admin.api.ConfigurationAdmin:service=ui",
             idAttribute: 'name',
-            defaults: function() {
-              return {
-                currentUrl:     undefined,
-                isLoopbackUrl:   undefined,
-              };
+            defaults: {
```

Comment:
❓ Not `function()`?

Reply:
None of the attribute values are objects, so the `function()` syntax isn't necessary. See https://stackoverflow.com/q/17318048